crates/iota-sdk-ffi/README.md

```diff
@@ -0,0 +1,21 @@
+# IOTA SDK Bindings
+
+This crate can generate bindings for various languages (Go, Kotlin, Python, etc.) using [UniFFI](https://github.com/mozilla/uniffi-rs).
+
+Start by building the library to generate the appropriate dylib files.
+
+```sh
+cargo build -p iota-sdk-ffi --lib --release
+```
+
+Next, run the binary to generate the bindings for the desired language.
+
```

Comment:
Please add a python section name so we can later add other languages clearly

Reply:
Let's do this in a later PR please

Reply:
Then please open an issue for it

Reply:
https://github.com/iotaledger/iota-rust-sdk/issues/75